lib/sql.ml

```diff
@@ -330,6 +330,7 @@ type stmt =
 | CreateIndex of string * table_name * string list (* index name, table name, columns *)
 | Insert of insert_action
 | Delete of table_name * expr option
+| DeleteMulti of table_name * table_name list * nested * expr option
```

Comment:
this table_name is not special, I understand it is ad hoc typed promise of not empty list, I would rather see one list